app/controllers/api/connections_controller.rb

```diff
@@ -122,10 +123,76 @@ def update_match
 
     end
 
+    def update_connection
+
+        if @connection.in_progress?
+
+            begin
+                @connection.class_status = params[:status]
+
+            rescue
+                render_json_400("Updated status is invalid")
+                return
```

Comment:
Didn't know rails still renders the response if you return nil. Nice! Nonetheless, you can also do:
```ruby
return render_json_400("Updated status is invalid")
```